portfolio/src/main/webapp/index.html

```diff
@@ -7,6 +7,13 @@
     <script src="script.js"></script>
   </head>
   <body>
+    <button onclick="topFunction()" id="toTop">Top</button>
+    <div class="tab">
```

Comment:
Not sure if you're aware that you can also use `<a href="#my_id">` to jump to sections of a given page.

https://marksheet.io/html-links.html#anchor-targets

Reply:
thats what i did at first, but it would place an underline under the tab names and would turn purple after being clicked once. I switched it to what I have now because of that.